js/frames/packM15HoloStamps.js

```diff
@@ -17,6 +18,14 @@ availableFrames = [
 	{name:'Gray Holo Stamp', src:'/img/frames/m15/holoStamps/gray.png', bounds:{x:0.4554, y:0.9172, width:0.0894, height:0.0320}},
 	{name:'Acorn Holo Stamp', src:'/img/frames/m15/holoStamps/acorn.png', bounds:{x:0.4554, y:0.9129, width:0.0894, height:0.0381}},
 	{name:'Alchemy Holo Stamp', src:'/img/frames/m15/holoStamps/alchemy.png', bounds:{x:0.4554, y:0.9172, width:0.0894, height:0.0320}},
+	{name:'JP White Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampW.png', bounds:boundsJP},
+	{name:'JP Blue Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampU.png', bounds:boundsJP},
+	{name:'JP Black Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampB.png', bounds:boundsJP},
+	{name:'JP Red Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampR.png', bounds:boundsJP},
+	{name:'JP Green Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampG.png', bounds:boundsJP},
+	{name:'JP Multicolored Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampM.png', bounds:boundsJP},
+	{name:'JP Artifact Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampA.png', bounds:boundsJP},
+	{name:'JP Land Holo Stamp', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseHolostampL.png', bounds:boundsJP},
 ];
```

Comment:
Instead of including the stamps here, they can just be added as part of the Japan Showcase frame, since they're only applicable to that one.